### ISSUE TITLE
Download new version

### ISSUE THREAD

**Issue by chWolfgang** [RedHatter/Graveyard#44]
Noob question:
I noticed that there are updates to the master, but no new download for over a year.
How do I install the new version?
Thanks for the help and for making the new version.

**Comment by Supergecki** [comment]
Hello there!
Normally, RedHatter would have to release a new version (because just he has write access).
If you want to use the new version now, I could send you a copy of the zip folder.
[Graveyard.zip](https://github.com/RedHatter/Graveyard/files/3317949/Graveyard.zip)

Now you can install it as you would do with other plugins.
I hope it works!

**Comment by chWolfgang** [comment]
TYVM!!
I don't have time to test it right now, but it seems to work. 
Is MASS RESURRECTION supported as a resurrection card?
Thanks again for your time. I love this addon!

**Comment by Supergecki** [comment]
No, Mass Resurrection isn't supported by now. I wonder whether Catrina Muerte should be added?
But thanks for the suggestion. Maybe I'll publish a version with those cards (and Witching Hour).

**Comment by chWolfgang** [comment]
I think adding all of those would be great. (But I don't have to do the work  :)  )
Did you see today's announcement? I think Resurrect is going to be around as mechanic for a while. There is going to be a new nutural legendary that will resurrect friendly cards that died this turn. Outside of the scope of this add-on but shows Blizz's interest in the mechanic.

**Comment by Supergecki** [comment]
Pull request is out, added Witching Hour and both RoS resurrection cards. View the pull request [here](https://github.com/RedHatter/Graveyard/pull/45).
